CMakeLists.txt

```diff
@@ -20,8 +20,11 @@ message(STATUS "cmake build type: ${CMAKE_BUILD_TYPE}")
 
 project(mavlink_sitl_gazebo VERSION 1.0.0)
 
-set(CMAKE_CXX_STANDARD 14)
```

Comment:
@TSC21 this line is gone?

Reply:
Yes because we have https://github.com/PX4/sitl_gazebo/pull/550/files#diff-af3b638bc2a3e6c650974192a53c7291R26.

Reply:
Ok, so now we require C++11 or higher.